core/templates/components/forms/custom-forms-directives/image-uploader.directive.ts

```diff
@@ -120,7 +126,12 @@ angular.module('oppia').directive('imageUploader', [
 
         $(elt).bind('dragover', function(e) {
```

Comment:
@Radesh-kumar  @kevintab95 instead of using jquery lite, can we use AngularJS provided event listeners for this? (I know that jQuery lite is provided by AngularJS, but this is not inline with Angular.

Reply:
Yep, I'm fine with that. But just to confirm do we actually use jQuery lite or jQuery here?

Reply:
I think both

Reply:
I guess (elt) is for JQlite and (e) for Jquery, my Vs code suggesting me otherwise

Reply:
On second thoughts, it looks like angularjs doesn't have such sort of event bindings for dragover. Angular has it in its component development kit. @Radesh-kumar this is no longer a conrern.

Reply:
> On second thoughts, it looks like angularjs doesn't have such sort of event bindings for dragover. Angular has it in its component development kit. @Radesh-kumar this is no longer a conrern.

@kevintab95 PTAL